nextcord/client.py

```diff
@@ -248,6 +248,10 @@ class Client:
         Warning: While enabling this will prevent "ghost" commands on guilds with removed code references, rolling out
         to ALL guilds with anything other than a very small bot will likely cause it to get rate limited.
 
+    rollout_guild_ids: List[:class:`int`]
+        Allow user to set a global scope of guild ids instead of filling in `guild_ids` parameter for each application
+        command decorator. Defaults to ``[]``
```

Comment:
This should be None?

Reply:
To be consistent with application commands, defaulting to empty list is fine.

It was possibly done in a bit of an non-intuitive manner, but currently setting `guild_ids=[]` on a command is the same as not setting it.